CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 * 21.4.1
+ - Added ISTA algorithm. Improve inheritance of proximal gradient methods.
```

Comment:
This is not in 21.4.1, it will be in 22.0.0. Move it up.

Reply:
Added.